src/seedu/addressbook/AddressBook.java

```diff
@@ -1213,14 +1213,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix from the given fullString if prefix occurs at the start of the string.
      *
      * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param prefix  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefixSign(String fullString, String prefix) {
+        return fullString.replaceFirst(prefix, "");
```

Comment:
Does this remove the prefix when it occurs at the beginning of fullString or does it just remove the first occurrence of prefix in fullString (could be in the middle)

Can abstract this further.